ledger/api-server-damlonx/reference-v2/src/main/scala/com/daml/ledger/api/server/damlonx/reference/v2/ReferenceServer.scala

```diff
@@ -46,8 +48,9 @@ object ReferenceServer extends App {
 
   config.archiveFiles.foreach { file =>
     val archivesTry = for {
-      zipFile <- Try(new ZipFile(file))
-      dar <- DarReader { case (_, x) => Try(Archive.parseFrom(x)) }.readArchive(zipFile)
+      zipInputStream <- Try(new ZipInputStream(new FileInputStream(file)))
```

Comment:
Again this should be just the call to `readArchive(darFile: ZipFile): Try[Dar[A]]` which should also take care of the closing the input stream, which are not doing here.